### PR TITLE
Add missing migrations

### PR DESCRIPTION
**Pull request by stsewd**
There are some missing migrations on the project, this make a little difficult to isolate migrations for other recent changes.

### CODE REVIEW

**Comment by humitos** [comment]
Good! I noticed this some time ago and I wasn't sure why this was happening. Do you know why these migrations are needed?

I supposed that it could be some `help_text` that changed or `choices` so a new migration is needed but in the migration files it seems there are `CreateModel` calls. So, I'm confused. How the project could work without those models created? Aren't those models necessary for the project? Should we remove them?

**Comment by safwanrahman** [comment]
So this models were added by #2764
@agjohnson Do you have any idea why the migration was not added? and how that is working now in production?

**Comment by stsewd** [comment]
@humitos @safwanrahman maybe something to do with the `python manage.py loaddata test_data` step (just trying to guess here)?

**Comment by humitos** [comment]
@stsewd the `loaddata` command doesn't affect the models, it just add data to the database without touching its structure. So, that shouldn't be the issue.

**Comment by humitos** [comment]
@ericholscher can you take a quick look at this PR? This is causing problems in other PRs to generate the proper migrations files.

**Comment by stsewd** [comment]
> but the integration proxy models might be either a missed migration or a difference in django versions or something?

@agjohnson I did a grep and didn't find any previous migration about that models on the project :/

I updated the names, I'm not sure if I chosen the more descriptive name for some migrations, let me know if you want me to change some names.

**Comment by ericholscher** [comment]
This looks good, and will remove the `need to run migration` warnings locally. Managing migration names is hard, but I'd say we merge it this week along w/ our other "cleanup" PR's, so we can test it locally and then deploy next week.

Will wait on @agjohnson's re-review to merge.

File: readthedocs/gold/migrations/0002_add_choices_for_level.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (humitos) [inline readthedocs/gold/migrations/0002_add_choices_for_level.py]: Name suggestion: `0002_alter_choices_in_level_model.py`

File: readthedocs/integrations/migrations/0003_create_webhook_models.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```
Comment (humitos) [inline readthedocs/integrations/migrations/0003_create_webhook_models.py]: Name suggestion: `0003_create_proxy_webhook_models.py`

File: readthedocs/projects/migrations/0024_alter_domain_model.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```
Comment (humitos) [inline readthedocs/projects/migrations/0024_alter_domain_model.py]: Name suggestion: `0024_alter_and_project_models.py`

Comment (stsewd) [reply readthedocs/projects/migrations/0024_alter_domain_model.py]: `...alter_and` domain and project, right?

**Comment by ericholscher** [comment]
This is outdated. Closing this, we should just run this right before doing a deploy, since any PR will likely conflict or go stale.

**Pull request closed without merging**